engine/src/main/java/org/terasology/engine/core/PathManager.java

```diff
@@ -33,15 +32,14 @@
  */
 public final class PathManager {
     private static final Logger LOGGER = LoggerFactory.getLogger(PathManager.class);
-    private static final String TERASOLOGY_FOLDER_NAME = "Terasology";
-    private static final Path LINUX_HOME_SUBPATH = Paths.get(".local", "share", "terasology");
-
+    private static final ProjectDirectories projDirs = ProjectDirectories.from("org", "terasology", "terasology");
```

Comment:
```suggestion
    private static final ProjectDirectories PROJECT_DIRS = ProjectDirectories.from("org", "terasology", "terasology");
```

Checkstyle is complaining about the naming style here. I also think "proj" is a bit vague for an abbreviated form of "project".